android/src/main/java/com/maskedtextinput/events/EventNames.kt

```diff
@@ -1,5 +1,5 @@
 package com.maskedtextinput.events
 
 object EventNames {
-  const val CHANGE_TEXT_EVENT = "changeText"
+  const val CHANGE_TEXT_EVENT = "onAdvancedMaskTextChange"
```

Comment:
Personally me - I am keeping this as a companion object for Event class. I think such approach better follows SOLID principles where you should have only one reason to change a file
If you keep all events in one class, then you have more reasons: you may add new event, you may remove old event, etc.